projects/igniteui-angular/src/lib/grids/summaries/summary-cell.component.html

```diff
@@ -12,7 +12,11 @@
 
                 <span class="igx-grid-summary__label" [title]="summary.label">{{ translateSummary(summary) }}</span>
                 <span class="igx-grid-summary__result" [title]="summary.summaryResult">
-                    {{ summary.summaryResult }}
+                    {{
+                        column.summaryFormatter
+                        ? (summary | summaryFormatter:column.summaries:column.summaryFormatter)
+                        : appySummaryPipe(summary)
```

Comment:
It is better to apply the pipes in the template so that we don't create the pipes each time in the code and also benefit from the built-in cache of the pipes.